### PR TITLE
[LAM-88]As a ~okeanos user, I want to be able to destroy a λ instance I own 

### PR DESCRIPTION
**Pull request by efikalti**
Added methods for deleting:
- cluster
- vm
- private network

In addition:
- provisioner reserves ips from the existing pool first and then creates new if needed
- in check_resources, there is one additional check if the flavor allows creation

### CODE REVIEW

**Comment by pkoro** [comment]
@efikalti can you rebase this on top of current devel?

**Comment by efikalti** [comment]
@pkoro please review

File: core/fokia/provisioner.py [hunk 1]
```diff
@@ -540,6 +602,8 @@ def check_all_resources(self, quotas, **kwargs):
                                           ip_request=args.ip_request,
                                           network_request=args.network_request,
                                           project_name=args.project_name)
-    # print(response)
-    # print(provisioner.get_cluster_details())
-    # print(provisioner.get_private_key())
+    print(provisioner.get_cluster_details())
```
Comment (pkoro) [inline core/fokia/provisioner.py]: Maybe remove this line.

Comment (efikalti) [inline core/fokia/provisioner.py]: I use this part of the code for local testing,it will all be removed in the future

**Comment by themiszamani** [comment]
could you please update the documentation?
How to 
a) create_lambda_cluster
b) delete_cluster 
c) get_cluster_details

**Comment by efikalti** [comment]
@themiszamani 
Updated the README to include the steps for creating/deleting a lambda cluster, note that deleting will not actually delete any cluster at this moment because there is no database to retrieve the necessary information from, yet.

**Pull request merged**